src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -44,9 +44,11 @@ public static class Vector64
         public static Vector64<uint> CreateScalarUnsafe(uint value) { throw null; }
     }
     [StructLayout(LayoutKind.Sequential, Size = 8)]
-    public readonly struct Vector64<T> where T : struct
+    public readonly struct Vector64<T> : IEquatable<Vector64<T>>, IFormattable
```

Comment:
What are the steps for auto-generating the ref? I'd like to run it and see what the delta is.

Reply:
For this case, since the source isn't coming from corefx, run the following from the ref directory:
`msbuild /t:GenerateReferenceSource`

See the documentation here for details: https://github.com/dotnet/corefx/blob/578b856794f77e4778b6a1b0584cc7c5b98a9465/Documentation/coding-guidelines/updating-ref-source.md

Please update it if you discover something else.